.yarnrc

```diff
@@ -4,4 +4,3 @@
 
 disable-self-update-check true
 lastUpdateCheck 1562193150687
```

Comment:
I guess we can remove this file altogether now. `disable-self-update-check true` is nice but not crucial.

Reply:
imo let's keep `disable-self-update-check`

Reply:
That's basically synonymous to pip's `--disable-pip-version-check`, yeah? I'd like to keep it too, saves a network call.